src/message/messageActionSheet.js

```diff
@@ -38,12 +38,19 @@ const muteStream = ({ auth, message, subscriptions }) => {
   muteStreamApi(auth, sub.stream_id);
 };
 
+const shareMessage = ({ message }) => {
+  Share.share({
+    message: message.content.replace(/<(?:.|\n)*?>/gm, '')
```

Comment:
Sharing content will also be useful for those who are not on that realm. (And one can easily get content without clicking link). We can add one more option for link. Slack have "Copy message link."